DaFont/dist/presence.js

```diff
@@ -0,0 +1,74 @@
+const presence = new Presence({
+    clientId: "685827254256795677",
+    mediaKeys: false
+});
+
+presence.on('UpdateData', () => {
+    const presenceData = {
+            largeImageKey: "dafont"
+        },
+        Path = document.location.pathname,
+        Details = {};
+    presenceData.startTimestamp = Math.floor(Date.now() / 1000)
```

Comment:
This is optional but for the "quality" of your code, you may want to move the `startTimestamp` directly into your `presenceData` object since you're not changing it anywhere else, there's no reason to set this outside of it. It's okay if you want to keep it but it'll save a line!

Reply:
^^

Reply:
Sure!